core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -143,6 +143,10 @@ private TableProperties() {
   public static final String DELETE_PARQUET_PAGE_SIZE_BYTES = "write.delete.parquet.page-size-bytes";
   public static final int PARQUET_PAGE_SIZE_BYTES_DEFAULT = 1024 * 1024; // 1 MB
 
+  public static final String PARQUET_PAGE_ROW_LIMIT = "write.parquet.page-row-limit";
+  public static final String DELETE_PARQUET_PAGE_ROW_LIMIT = "write.delete.parquet.page-row-limit";
```

Comment:
what about `page-row-count-limit`?

Just `row-limit` can also be interpreted as row size/bytes limit.

Reply:
That's what I had originally (and how it is defined in parquet-mr), @rdblue preferred this naming. I'm OK with either one.

Reply:
sorry. missed the earlier context. `page-row-count-limit` is a little more clear to me.

Reply:
I think it is pretty clear from context that this is the number of rows and not the size of a row. For these formats, types can be size limited, but rows aren't.

Reply:
I think `page-row-limit` is clear enough.  There is also a page size limit, but I feel the count is implied given it's rows.